Machine Learning/Hands-on Implementation using sklearn/Linear Regression/readme.md

```diff
@@ -0,0 +1,13 @@
+# Linear Regression with scikit-learn in Python
+
+Implemented Using: Jupyter Notebook
```

Comment:
Could you add the dataset you use for this example with credits?

Reply:
Actually, I had got this dataset from someone else quite a long time ago. I don't have any links for this. Would it be fine?

Reply:
@Aditya-1500 Here is the dataset source - https://www.kaggle.com/srolka/ecommerce-customers You could maybe add-

We use the [Ecommerce Customers Dataset](https://www.kaggle.com/srolka/ecommerce-customers) originally contributed by [Steven Rolka](https://www.kaggle.com/srolka)

Reply:
Okay thanks. Will add this as well.